data/json/items/book/survival.json

```diff
@@ -194,6 +193,7 @@
     "looks_like": "textbook_firstaid",
     "skill": "survival",
     "required_level": 3,
+    "proficiencies": [ { "proficiency": "prof_chitinworking", "time_factor": 0.85, "fail_factor": 0.75 } ],
```

Comment:
I am not sure a textbook on arthropods would reasonably be helpful at working giant mutant bug chitin at all, but if it was it would be of very little benefit at all and definitely wouldn't almost give you the proficiency outright.

Reply:
Changed to give a minimum impact (but still, better than nothing, these books help to obtain the chitin recipes, they should have some useful info).

The values were inverted before because of my poor understanding of the modifiers.